product_docs/docs/tde/15/enabling_tde.mdx

```diff
@@ -108,3 +109,17 @@ This example uses EDB Postgres Advanced Server 15 running on a Linux platform. I
    data_encryption_key_unwrap_command = 'openssl enc -d -aes-128-cbc -pass pass:ok -in %p'
    ```
 
+## Checking for TDE in use using SQL
```

Comment:
Can we reword this like - Checking for TDE presence using SQL. or something else? Intend is not to use "use" along with using.

Reply:
Done